packages/react/src/badge/Badge.css.ts

```diff
@@ -0,0 +1,92 @@
+import { theme } from "../styles";
+import {
+  type RecipeVariants,
+  createVar,
+  recipe,
+  style,
+} from "../vanilla-extract";
+
+const subtleBackgroundColorVar = createVar();
+const subtleColorVar = createVar();
+const accentBackgroundColorVar = createVar();
+const accentColorVar = createVar();
+
+export const badge = recipe({
+  base: [
+    {
+      alignItems: "center",
+      fontWeight: "500",
+      justifyContent: "center",
+      px: "8",
+      py: "2",
+      rounded: "sm",
```

Comment:
radius is `xl` according to figma

Reply:
For badge, added the `md` rounding, for indicator will give the `xl` rounding vai.